sdks/wasm/Mono.WebAssembly.DebuggerProxy/InspectorClient.cs

```diff
@@ -8,13 +8,16 @@
 using System.IO;
 using System.Text;
 using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
 
 namespace WebAssembly.Net.Debugging {
-	public class InspectorClient : DevToolsClient {
+	internal class InspectorClient : DevToolsClient {
```

Comment:
InspectorClient is used in DebuggerTestSuite but we could move it and DevToolsClient out of this assembly

Reply:
Good point, thanks. Fixed.

IVT seems like an appropriate solution here since the API is being exposed purely for tests, so added that.